test/Components.re

```diff
@@ -221,7 +222,8 @@ module EmptyComponentWithAlwaysEffect = {
 };
 
 module EmptyComponentWithOnMountEffect = {
-  let%component make = (~onEffect, ~onEffectDispose, ()) => {
+  let%component make =
+                (~onEffect: unit => unit, ~onEffectDispose: unit => unit, ()) => {
```

Comment:
Why are this type annotation needed ?

Reply:
It complained about not being able to generalize a type, but this is only used in tests so it doesn't bleed anywhere.

Reply:
weird!